src/libs/prefs.js

```diff
@@ -3,6 +3,9 @@ import { getDynamic, setDynamic } from 'src/libs/browser-storage'
 
 
 const withUserPrefix = key => `${getUser().id}/${key}`
+const withUserPrefixForSpecifiedUserId = (userId, key) => `${userId}/${key}`
 
 export const getLocalPref = key => getDynamic(localStorage, withUserPrefix(key))
+// Needed for loading from localStorage on signin or signout event
+export const getLocalPrefForUserId = (userId, key) => getDynamic(localStorage, withUserPrefixForSpecifiedUserId(userId, key))
```

Comment:
I'm pretty sure it's safe, but just to be 100% sure, it's ok to give out the user ID right? (a one word answer is totally fine)

Reply:
It is no more unsafe than what we are already doing. (since other storage mechanisms are also using that user information) I believe it is anonymous though.